index.html

```diff
@@ -594,8 +605,11 @@ <h2>Forgot Password</h2>
 
                             <div class="card">
                                 <div class="overflow-img">
-                                    <a href="blog.html">
-                                        <img style="height: 250px;width:300px" src="./assets/img6.jpg" class="img-fluid">
+
+                                    <a href="#">
+                                        <img style="height: 250px;width:300px" src="./assets/img6.jpg" class="img-fluid" alt="Tech travels">

```

Comment:
Add blog.html in href don't add #

Reply:
@hiteashgupta1 changes are done, kindly check it out and merge